packages/widgets/src/lib/components/InferenceWidget/shared/WidgetExamples/WidgetExamples.svelte

```diff
@@ -49,13 +49,19 @@
 	}
 
 	function _applyWidgetExample(idx: number) {
+		if (!isOptionsVisible) {
+			return;
+		}
 		hideOptions();
 		const sample = examples[idx];
 		title = sample.example_title as string;
 		applyWidgetExample(sample);
 	}
 
 	function _previewInputSample(idx: number) {
+		if (!isOptionsVisible) {
+			return;
+		}
```

Comment:
needed those because of https://github.com/huggingface/huggingface.js/blob/13b95853a57718794b1bd21ccb522474a9a656b3/packages/widgets/src/lib/components/InferenceWidget/shared/WidgetExamples/WidgetExamples.svelte#L158

this `transition:slide`, when a user clicks & while options were being closed, https://github.com/huggingface/huggingface.js/blob/13b95853a57718794b1bd21ccb522474a9a656b3/packages/widgets/src/lib/components/InferenceWidget/shared/WidgetExamples/WidgetExamples.svelte#L165 was still being trigerred

Reply:
nice catch